packages/contracts/contracts/LenderCommitmentForwarder.sol

```diff
@@ -16,6 +23,12 @@ contract LenderCommitmentForwarder is TellerV2MarketForwarder {
         uint32 expiration;
         uint32 maxDuration;
         uint16 minInterestRate;
+        address collateralTokenAddress;
+        uint256 maxPrincipalPerCollateralAmount; //zero means infinite
+        CollateralType collateralTokenType; //erc721, erc1155 or erc20
+        address lender;
+        uint256 marketId;
+        address principalTokenAddress;
```

Comment:
should be a borrower field so a commitment can be restricted to a particular borrower. ZeroAddress is wildcard.